dcli/language/kotlin/types.json

```diff
@@ -0,0 +1,105 @@
+{
+  "Byte": {
+    "requireImport": false,
+    "className": "kotlin.Byte",
+    "defaultValue": 0,
+    "primitive": false
+  },
+  "byte": {
```

Comment:
You cannot use primitive type in Kotlin, same for short, long, int, etc.

Reply:
done